.github/workflows/pre-release-workflow.yml

```diff
@@ -80,45 +75,22 @@ jobs:
           name: android-sdk-test-results
           path: embrace-android-sdk/build/reports/tests/
 
-      - name: Configure git
-        run: |
-          git config --global user.name 'embrace-ci[bot]'
-          git config --global user.email 'embrace-ci@users.noreply.github.com'
-          git config --global url."https://${{ secrets.CD_GITHUB_USER }}:${{ secrets.CD_GITHUB_TOKEN }}@github.com".insteadOf "https://github.com"
-
-      - name: Gradlew Pre-Release
-        run: |
-          sed -i -r "s#version = ([^\']+)#version = ${{ github.event.inputs.version_to_release }}.0#" gradle.properties
-          git add gradle.properties
-          git commit -m "CI/CD: change version to be released: ${{ github.event.inputs.version_to_release }}.0"
-          git push
-          ./gradlew clean publishReleasePublicationToSonatype closeSonatypeStagingRepository -Dorg.gradle.parallel=false --stacktrace
-
-      - name: Set next SDK version
-        run: |
-          git checkout master
-          sed -i -r "s#version = ([^\']+)#version = ${{ github.event.inputs.next_version }}-SNAPSHOT#" gradle.properties
-          git add gradle.properties
-          git commit -m "CI/CD: set next version: ${{ github.event.inputs.next_version }}"
-          git push
-
       - name: Checkout Swazzler
         uses: actions/checkout@v3
         with:
           repository: embrace-io/embrace-swazzler3
           ref: master
-          path: ./embrace-swazzler3
           token: ${{ secrets.CD_GITHUB_TOKEN }}
 
-      - name: Swazzler Release
+      - name: Swazzler Pre-Release - Publish and Close repository
         run: |
-          cd ./embrace-swazzler3
-          ./gradlew clean release -Dorg.gradle.parallel=false -Prelease.useAutomaticVersion=true -Prelease.pushReleaseVersionBranch=release/${{ github.event.inputs.version_to_release }} -Prelease.releaseVersion=${{ github.event.inputs.version_to_release }}.0 -Prelease.newVersion=${{ github.event.inputs.next_version }}-SNAPSHOT --stacktrace
+          sed -i -r "s#version = ([^\']+)#version = ${{ github.event.inputs.current_version }}#" gradle.properties
```

Comment:
Third parameter of `sed` needs to start off with `s#version=` instead of `s#version =` (notice the removal of the space before the `=` sign. This is because the file doesn't have that space - this is likely why the version was screwed up in the swazzler.

Alternatively, we can just modify the swazzler properties file to add that space so this replacement will work. Or we can make the regular expression a bit more complicated to take care of both cases. Either way, we need to change.